app/src/main/res/layout/activity_main.xml

```diff
@@ -52,15 +52,16 @@
         android:layout_height="49dp"
         android:autofillHints="name"
         android:background="@drawable/inputs_login"
+        android:fontFamily="@font/roboto_light_italic"
```

Comment:
you can create a custom style that includes fontFamily, textSize, textColor and textStyle and use it wherever you need in ones single line of code

Reply:
thanks